.appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   # Also cannot be set because R uses its compile-time settings.
   - ps: sed -i 's|CXX_STD = CXX14||' src/Makevars.win
   - conda config --set always_yes yes --set changeps1 no
-  # - conda update -q conda
```

Comment:
This was previously commented out to prevent the update to a buggy version of conda.